CHANGELOG.md

```diff
@@ -21,8 +21,8 @@
   * Fix issue projects would fail to load with missing output path error (PR: [#74791](https://github.com/dotnet/roslyn/pull/74791))
   * Expose option to disable completion triggers argument list (PR: [#74792](https://github.com/dotnet/roslyn/pull/74792))
 * Update Debugger to v2.43.0 (PR: [#7420](https://github.com/dotnet/vscode-csharp/pull/7420))
-* Bump xamltools to 17.12.35223.16 (PR: [#7464](https://github.com/dotnet/vscode-csharp/pull/7464))
 * Added XAML Hot Reload support for x:FactoryMethod and x:Arguments
+* Bump xamltools to 17.12.35227.331 (PR: [#7489](https://github.com/dotnet/vscode-csharp/pull/7489))
```

Comment:
This changelog looks out of date compared to main - you might need to pull and move this to a new section
https://github.com/dotnet/vscode-csharp/blob/main/CHANGELOG.md